Directory.Build.props

```diff
@@ -39,12 +39,13 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(BuildFlavor)' == 'Current' ">
-    <DefineConstants>$(DefineConstants);NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
-    <TargetFramework>netstandard1.5</TargetFramework>
-    <TestProjectTargetFramework>net462</TestProjectTargetFramework>
+    <DefineConstants>$(DefineConstants);BUILDFLAVOR_CURRENT;USE_EVENTHUB</DefineConstants>
```

Comment:
If we are already changing these, can we do the opposite? Let's have the Current build flavor not add any additional conditional compilation, and instead only add additional ones if we are building the Legacy flavor, which will eventually be killed

Reply:
We can do, my thinking was to minimize code changes. For the ServiceBus assembly we need both branches of the #ifs and we'll removed those parts later as you already mentioned.
The idea behind BUILDFLAVOR_CURRENT is to make the previously used NETSTANDARD, NETSTANDARD_TODO cleanup easier in the future, since now we're not depending on those.

@jdom, please advise on what would you like to see.

Reply:
but removing these would have the same amount of changes (although negating the conditional). So `#if NETSTANDARD` would be replaced with `#if !BUILD_FLAVOR_LEGACY` instead of `#if BUILD_FLAVOR_CURRENT`. So same amount of changes.
With event hub, not sure I understand why is that better than just using the build flavor conditional compilation, but at first look I think it would be preferred that it reads BUILD_FLAVOR_LEGACY

Reply:
what does USE_EVENTHUB param do ?

Reply:
@xiazen looks like it define to ServiceBus old package or new Eventhub netstandard one.